src/types/restrictions.d.ts

```diff
@@ -1,3 +1,5 @@
+import bodyParser from "body-parser";
+
```

Comment:
Hey, mind removing this import? Looks like it might have gotten added by mistake.